packages/expo-camera/src/Camera.tsx

```diff
@@ -141,13 +151,12 @@ export default class Camera extends React.Component<CameraProps> {
     return await CameraManager.getAvailablePictureSizes(ratio, this._cameraHandle);
   }
 
-  async recordAsync(options?: CameraRecordingOptions): Promise<{ uri: string }> {
+  async recordAsync(options?: CameraRecordingOptions): Promise<{ uri: string; codec: VideoCodec }> {
```

Comment:
```suggestion
  async recordAsync(options?: CameraRecordingOptions): Promise<{ uri: string }> {
```
I don't think we need to return this value here 🤔 
1. Use provide it explicitly, so it's known already
2. Default one is described in the docs? 🤔